File: golang/speedtest.go
package main

import (
	"fmt"
	"time"
)

func main() {
	start := time.Now()
	number := 50000000	

	for i := 0; i < number; i++ {
	}

	elapsed := time.Since(start)
	fmt.Printf("Counting to %d in Go took %s\n",number , elapsed)
}

File: rust/speedtest.rs
use std::time::Instant;

fn main() {
    let start = Instant::now();

    let mut count = 0;
    let number = 50_000_000;

    for _i in 0..number {
        count += 1;
    }

    let duration = start.elapsed();

    println!("Counting to {} in Rust took {:?} ms.", number, duration.as_millis());
}
